controller/codebase.go

```diff
@@ -264,7 +264,7 @@ func getNamespace(ctx context.Context) string {
 	token := goajwt.ContextJWT(ctx)
 	if claims, ok := token.Claims.(jwt.MapClaims); ok {
 		email := claims["email"].(string)
-		return strings.Split(email, "@")[0] + "-dsaas-che"
+		return strings.Replace(strings.Split(email, "@")[0], ".", "-", -1) + "-che"
```

Comment:
Shouldn't we pass this `-che` or `-dsaas-che` prefix as an ENV variable ?  Or maybe even asking to che services to offer an API endpoint or something to avoid guessing it.